news/2024/2024-06-27-osu-catch-world-cup-2024-finals-recap.md

```diff
@@ -0,0 +1,98 @@
+</div>
+
+The mappool has reached a difficulty where a full-combo on any of the maps is becoming challenging for some and impossible for many. We saw a handfull of FCs on [HD3](https://osu.ppy.sh/beatmapsets/2198816#fruits/4653147), [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099) and [HR3](https://osu.ppy.sh/beatmapsets/2198812#fruits/4653138).
+
+The most picked maps in the pool this week are [NM4](https://osu.ppy.sh/beatmapsets/2198872#fruits/4653265) and [MX2](https://osu.ppy.sh/beatmapsets/2198798#fruits/4653117) whilst [NM1](https://osu.ppy.sh/beatmapsets/2198766#fruits/4653068) and [DT2](https://osu.ppy.sh/beatmapsets/2198875#fruits/4653272) were never picked.
+
+The mappool selection team is treating the players this year with yet another osu! original coming out for this mappool. The Mixed Mod 3 slot is filled by Jemzuu's [Neverlast](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125), a song produced by Aoi.
+
+[![](/wiki/shared/news/2024-06-27-osu-catch-world-cup-2024-finals-recap/neverlast.jpg)](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/183/Songs/Aoi_-_Neverlast.mp3">
+    </audio>
+</div>
+
+## Finals results
+
+We started the weekend on saturday with two loser bracket matches, where Poland, Chile, Finland and Canada fought it out to advance one match futher and secure atleast a top 4 finish this year. 
+
+On sunday, Argentina and the United States faced off in the winners finals. The United States have only lost a single map in the entire tournament up until now. Argentina on the other hand has had multiple close encounters and made it to winners finals being the underdogs in most of their recent matches.
+
+The weekend ended with the winners on saturday battling it out to reach the grand finals.
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| Chile ::{ flag=CL }:: | 6 | **7** | ::{ flag=PL }:: **Poland** |
+| Finland ::{ flag=FI }:: | 1 | **7** | ::{ flag=CA }:: **Canada** |
+| **United States** ::{ flag=US }:: | **7** | 1 | ::{ flag=AR }:: Argentina |
+| Canada ::{ flag=CA }:: | 5 | **7** | ::{ flag=PL }:: **Poland** |
+
+## Match highlights
+
+### ::{ flag=CL }:: Chile vs. ::{ flag=PL }:: Poland
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2179700308&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Both Chile and Poland have looked very strong during this years osu!catch world cup, so the stage was set and the expectations high as these two titans faced off to secure themselves a top 4 finish. Chile is the final remaining country that has previously won the world cup, albeit this being 10 years ago already.
```

Comment:
I think it's fine to use the abbreviation at this point since "osu!catch World Cup" was used like 2-3 times already previously in this text
```suggestion
Both Chile and Poland have looked very strong during this year's CWC, so the stage was set and the expectations high as these two titans faced off to secure themselves a top 4 finish. Chile is the final remaining country that has previously won the world cup, albeit this being 10 years ago already.
```